lib/openstudio/extension/runner.rb

```diff
@@ -99,12 +100,12 @@ def initialize(dirname = Dir.pwd, bundle_without = [], options = {})
 
             # check existing config
             needs_config = true
-            if File.exist?('./.bundle/config') # checking wrt gemfile_dir
-              puts 'config exists'
+            if conf_bpath = Bundler.configured_bundle_path.explicit_path
+              puts 'bundler config exists'
               needs_config = false
-              config = YAML.load_file('./.bundle/config')
 
-              if config['BUNDLE_PATH'] != @bundle_install_path
+              if conf_bpath != @bundle_install_path
+                raise "Detected mistmatch between bundle's configured path #{conf_bpath} and runner configuration #{@bundle_install_path}"
```

Comment:
Now that it respects the configured path, just throw if they don't match. User will have to either trash the runner.conf or adjust it.